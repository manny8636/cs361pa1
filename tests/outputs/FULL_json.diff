--- outputs/FULL_json.txt	2022-02-15 14:30:54.527066372 -0500
+++ expected/FULL_json.txt	2021-01-25 15:27:24.000000000 -0500
@@ -0,0 +1,5 @@
+Key-value pairs:
+{
+KEYS[keyName] = string value
+}
+Success!
