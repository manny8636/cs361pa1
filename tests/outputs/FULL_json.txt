Parsing ../data/json.txt failed
