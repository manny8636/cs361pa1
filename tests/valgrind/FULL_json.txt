==372154== Memcheck, a memory error detector
==372154== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==372154== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==372154== Command: ../json -o ../data/json.txt
==372154== 
==372154== 
==372154== HEAP SUMMARY:
==372154==     in use at exit: 6,192 bytes in 3 blocks
==372154==   total heap usage: 6 allocs, 3 frees, 14,976 bytes allocated
==372154== 
==372154== LEAK SUMMARY:
==372154==    definitely lost: 6,192 bytes in 3 blocks
==372154==    indirectly lost: 0 bytes in 0 blocks
==372154==      possibly lost: 0 bytes in 0 blocks
==372154==    still reachable: 0 bytes in 0 blocks
==372154==         suppressed: 0 bytes in 0 blocks
==372154== Rerun with --leak-check=full to see details of leaked memory
==372154== 
==372154== For lists of detected and suppressed errors, rerun with: -s
==372154== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
