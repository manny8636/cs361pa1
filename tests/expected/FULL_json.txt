Key-value pairs:
{
KEYS[keyName] = string value
}
Success!
